share/spice/untappd/footer.handlebars

```diff
@@ -0,0 +1,7 @@
+<div class="untappd__footer tile__update__info two-line">
+    <span>{{abv}}% ABV</span>
+    <i class="sep"></i>
+    <span>{{ibu}} IBU</span>
+    <i class="sep"></i>
+    <span><i class="count__i"></i> {{ratings}}</span>
```

Comment:
@urbanserj do we have access to the rating as well? It seems like it would be best to show the rating along with the number of reviews?